papers/drivendata/main.md

```diff
@@ -300,10 +308,18 @@ Model performance tends to be lower for:
   - Heavily occluded or distant animals, or those only partially visible or present for just a few frames
   - Low-quality media, such as videos that are washed out or too dark
 
+Due to a lack of standard benchmarks or comparable tools that perform species classification for videos, it is difficult to provide a direct comparison to other results.[^footnote-video-benchmarking] As an indirect comparison, the SpeciesNet model used by Wildlife Insights for _image_ species classification reports an 81.9% accuracy on their evaluation dataset[@doi:10.1049/cvi2.12318]. This demonstrates that Zamba's pretrained model performance is in a comparable range to other published results on a separate-but-related task.
```

Comment:
this doesn't feel like a fair comparison since images are _substantially_ easier. i think we might want to remove here